packages/utils/src/TestUtils.ts

```diff
@@ -52,6 +52,24 @@ class TestUtils {
     fn: (...args: TArgs) => TResult
   ): jest.Mock<TResult, TArgs> => (fn as unknown) as jest.Mock<TResult, TArgs>;
 
+  /**
+   * Selectively disable logging methods on `console` object. Uses spyOn so that
+   * changes will be reverted after leaving the test scope that it is set in. If
+   * no method names are given, all will be disabled.
+   */
+  static disableConsoleOutput = (
+    ...methodNames: ('log' | 'warn' | 'error' | 'info' | 'debug')[]
+  ): void => {
```

Comment:
This will let you actually disable all console methods if you wanted to (there are 24 methods on console)

```suggestion
  static disableConsoleOutput(
    methodNames: (keyof typeof console)[]
  ): void {
```

Reply:
I did actually look at just using the `keyof Console`, but there's 1 prop that it doesn't like, so end up having to do `keyof Omit<Console, 'Console'>`.

Reply:
I was able to get this working